roles/deploy/tasks/providers/aws.yaml

```diff
@@ -28,7 +28,7 @@
   when: configuration.customer_gateway is defined
 
 - name: "Create VPN Connection"
-  community.aws.ec2_vpc_vpn:
+  amazon.aws.ec2_vpc_vpn:
```

Comment:
any reason for this change ?

Reply:
ansible-lint gives a warning saying use a canonical module `amazon.aws.ec2_vpc_vpn` instead of `community.aws.ec2_vpc_vpn`